src/jquery.multiselect.filter.js

```diff
@@ -149,9 +149,9 @@
         });
 
         // trigger the change event on the select
-        if($inputs.length)
+        if(inputCount)
```

Comment:
This is referencing the variable from that for loop above, you should move the declaration to outside the for loop if you are going to use it elsewhere.

Reply:
inputCount is defined at the top of the function and used in two places, one of them being in the for loop and one of them being here.

Reply:
**CORRECTION:** For some reason I did not realize you were referencing the jquery.multiselect.filter.js file here, even though GitHub clearly states that.  ;-D   Sorry... I changed this in another commit.